<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="jslave-traditional-v2-10-25-196-164" name="com.mypurecloud.sdk.v2.ApiClientRetryTest" tests="12" failures="0" timestamp="06 Jan 2021 04:42:30 GMT" time="69.338" errors="0">
  <testcase name="invokeTestWith_503" time="40.032" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_502" time="13.035" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_502_And_0_MaxRetryTime" time="0.002" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_504_No_MaxRetryTime" time="0.030" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_502" time="1.002" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_429_And_No_MaxRetryTime" time="0.034" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_503_And_RetryConfig" time="3.100" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_429" time="1.003" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_504" time="2.033" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_200" time="0.001" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_504_And_No_RetryAfter_Header" time="3.001" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_429" time="6.065" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
</testsuite> <!-- com.mypurecloud.sdk.v2.ApiClientRetryTest -->
